test/extended/util/cluster/cluster.go

```diff
@@ -128,9 +137,10 @@ func LoadConfig(state *ClusterState) (*ClusterConfiguration, error) {
 	zones.Delete("")
 
 	config := &ClusterConfiguration{
-		MultiMaster: len(state.Masters.Items) > 1,
-		MultiZone:   zones.Len() > 1,
-		Zones:       zones.List(),
+		MultiMaster:           len(state.Masters.Items) > 1,
```

Comment:
@smarterclayton @wking

Reply:
Usually the convention is to disable upstream tests in [o/k](https://github.com/openshift/kubernetes/blob/master/openshift-hack/test-kubernetes-e2e.sh) and then vendor o/o. This ensures we have single source of truth but in this case it could be ok considering `SingleReplicaTopology` var is defined here.